packages/web/src/app/Layout.tsx

```diff
@@ -1,34 +1,25 @@
-import { GetLanguagesResponseBody, Language } from '@translation/api-types';
 import { Suspense, useMemo } from 'react';
-import { Outlet, useLoaderData, useOutletContext } from 'react-router-dom';
-import apiClient from '../shared/apiClient';
+import { Outlet, useOutletContext } from 'react-router-dom';
 import useLocalStorageState from '../shared/useLocalStorageState';
 import Header from './Header';
 
-export function layoutLoader() {
-  return apiClient.languages.findAll();
-}
-
 export interface LayoutContext {
-  language?: Language;
+  language: string;
```

Comment:
I had to simplify how the selected translation language is selected to guarantee that a language is selected. Otherwise it adds complexity to the translation views if there is not a selected language